ui/app/components/app/home-notification/home-notification.container.js

```diff
@@ -0,0 +1,14 @@
+import { connect } from 'react-redux'
+import { withRouter } from 'react-router-dom'
+import { compose } from 'recompose'
+import BackupNotification from './home-notification.component'
+
+const mapDispatchToProps = () => {
```

Comment:
Is this empty mapDispatchToProps necessary?

Reply:
It is not, good catch!